frameworks/cassandra/universe/config.json

```diff
@@ -205,7 +205,11 @@
                 }
 
             }
-        }
+        },
+        "custom_domain": {
+          "type": "string",
+          "description": "A custom domain to be used in place of autoip.dcos.thisdcos.directory. This can be used to expose the service securely outside of the cluster, but requires setting up external DNS. See the service documentation for details."
+        },
```

Comment:
can you put this through a json linter? seems like formatting is off in multiple places.

Reply:
Fixed